### DISCUSSION TITLE
Better user signin & signup through loading button

### DISCUSSION THREAD

**Discussion by Gulshanaggarwal** [ykdojo/defaang#174] [Ideas]
**What's the idea?**
There should be a loading icon on the button when user presses signin & signup button. It will inform to them that it's performing the action otherwise, they will have to wait. It may confuse if there is not a loading.  It will improve UX definitely.

@ykdojo @subhoghoshX anything you would like to add? Can I create a new issue and start working on it?

**Comment by ykdojo** [comment]
Can you link the original issue where it was discussed?

**Reply by Gulshanaggarwal** [reply]
Yes, this is the link -  https://github.com/ykdojo/defaang/issues/140#issuecomment-1242924092

**Reply by Gulshanaggarwal** [reply]
Hey @ykdojo what's the plan?

**Reply by ykdojo** [reply]
this was my thought: https://github.com/ykdojo/defaang/issues/140#issuecomment-1244034529

**Reply by Gulshanaggarwal** [reply]
@ykdojo I got your point. I'm gonna put a loading button on signin & signup, rather than whole page. Should I start working on it?

**Reply by ykdojo** [reply]
Honestly I don't think it's a priority. Is there any other issue you'd be interested in working on?

**Reply by Gulshanaggarwal** [reply]
Ok @ykdojo  if no one is working on this one - [#108](https://github.com/ykdojo/defaang/issues/108) ,please assign it to me.

**Reply by ykdojo** [reply]
@Gulshanaggarwal we have a few ongoing discussions/PRs that are / might be relevant to that issue.

https://github.com/ykdojo/defaang/pull/137

https://github.com/ykdojo/defaang/pull/170

Mind taking a look at them first?

**Reply by ykdojo** [reply]
Also [this one](https://github.com/ykdojo/defaang/pull/138) and [this](https://github.com/ykdojo/defaang/discussions/150).
